src/entity/posts/utils.ts

```diff
@@ -298,7 +309,7 @@ export const createExternalLink = async ({
       createdAt: new Date(),
       sourceId: UNKNOWN_SOURCE,
       url,
-      canonicalUrl: url,
+      canonicalUrl: canonicalUrl || url,
```

Comment:
If I understand you correctly we are changing where we save clean and not clean URL, which is not how we did it before right?

Reply:
That is correct. Previously we stored the URL without any params in both columns. Now we stored the filtered one in `url`.

And the one without params in canonicalUrl, which is how we did previously.

This is so that users can actually access correct and filtered URL (in cases like Medium Friend links).

Reply:
I understand the reason, just think we might have some other cases where it works becaues of above, and now it will no longer work. Let's add tests and then evaluate again, probably check with Ido as well (if you did not already).

Reply:
Approval from Ido, so guess it's OK?

Reply:
Could be that Ante has edge cases that I missed in mind

Reply:
I don't have any specific in mind, just the change of how we store URLs sounds it could mess something 🤷 

Asked Denis and from Yggdrasil side it should be fine since he mostly takes cleaned URL from API and only removes utm_ params. 

So let's try it and monitor 👍

Reply:
Yeah, also checked with Denis on that regard